pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/thread/ThreadRuntimeFactory.java

```diff
@@ -79,14 +79,14 @@ public ThreadRuntimeFactory(String threadGroupName, String pulsarServiceUrl, Str
                                 String pulsarWebServiceUrl) throws Exception {
         initialize(threadGroupName, Optional.empty(), pulsarServiceUrl, authConfig,
                 storageServiceUrl, null, secretsProvider, collectorRegistry, narExtractionDirectory,
-                rootClassLoader, exposePulsarAdminClientEnabled, pulsarWebServiceUrl, Optional.empty());
+                rootClassLoader, exposePulsarAdminClientEnabled, pulsarWebServiceUrl, Optional.empty(),1);
```

Comment:
Why are we setting the number of listener threads here to be one?  I think this config should also be respected with using the other runtimes as well.

Reply:
This is not resolved